src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -186,6 +189,38 @@ def sort_code_mapping_configs(
                     sorted_configs.insert(0, config)
         return sorted_configs
 
+    def get_commit_sha(
```

Comment:
```suggestion
    def get_commit_sha_from_blame(
```
I would name it like this.